### ISSUE TITLE
Preview of the results

### ISSUE THREAD

**Issue by mhoehle** [rgieseke/opencoviddata#1]
Thanks for providing the code for generating the ageclass-time incidence matrices!

As a small suggestion: would it be possible to add a screenshot/image of the result somewhere in the read.me, so one knows what to expect from this repository? I tried to see if its possible to use https://htmlpreview.github.io/?https://github.com/rgieseke/opencoviddata/blob/main/docs/baden-w%C3%BCrttemberg.html directly, but this didn' work.

Thanks for providing open-source software!

**Comment by rgieseke** [comment]
Good point! It's rendered here: https://rgieseke.github.io/opencoviddata/ but i haven't yet added any links to the Readme or landing page, will do!
